go/vt/wrangler/materializer.go

```diff
@@ -497,8 +497,8 @@ func (wr *Wrangler) prepareCreateLookup(ctx context.Context, keyspace string, sp
 	}
 
 	// Validate input table
-	if len(specs.Tables) != 1 {
-		return nil, nil, nil, fmt.Errorf("exactly one table must be specified in the specs: %v", specs.Tables)
+	if len(specs.Tables) < 1 || len(specs.Tables) > 2 {
+		return nil, nil, nil, fmt.Errorf("one or two table must be specified in the specs: %v", specs.Tables)
```

Comment:
This is not a terribly helpful error message, IMO. What about something ~:
```
return nil, nil, nil, fmt.Errorf("you must specify the source table and optionally the target lookup table with the specific vindex of choice: %v", specs.Tables)
```

Reply:
Here we should validate that the if two tables are specified then one of them is the lookup table it self and throw the error accordingly.